helix-core/src/main/java/org/apache/helix/util/InstanceValidationUtil.java

```diff
@@ -120,17 +121,17 @@ public static String getInstanceHelixDisabledType(HelixDataAccessor dataAccessor
    * @param clusterConfig
```

Comment:
Maybe `getInstanceHelixDisabledType` need change as well. We probably don't want to read from cluster config here?

Reply:
I don't get it. Method `getInstanceHelixDisabledType` is reading clusterConfig for disabled type. How can we not read from cluster config?

Reply:
No optimization as this needs to be pure revert.

Reply:
This function `getInstanceHelixDisabledType ` is added in change #2006 that enables batch. We could choose to remove the whole function?

Reply:
removed the method and its use cases